sros2/sros2/verb/generate_policy.py

```diff
@@ -66,22 +66,32 @@ def get_policy(self, policy_file_path):
         if os.path.isfile(policy_file_path):
             return load_policy(policy_file_path)
         else:
-            profiles = etree.Element('profiles')
+            enclaves = etree.Element('enclaves')
             policy = etree.Element('policy')
             policy.attrib['version'] = POLICY_VERSION
-            policy.append(profiles)
+            policy.append(enclaves)
             return policy
 
     def get_profile(self, policy, node_name):
-        profile = policy.find(
+        enclave = policy.find(
+            path='enclaves/enclave[@path="{path}"]'.format(
+                path=node_name.path))
```

Comment:
Consider f-strings here.

Reply:
formatted this and similar strings of this file in https://github.com/ros2/sros2/pull/203/commits/cab6123d7dafe8a9d16e48266c229bb884960346